library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt

```diff
@@ -58,6 +62,40 @@ object BaristaListInteractions {
         )
     }
 
+    @JvmStatic
+    fun assertListItemCount(@IdRes listId: Int, expectedItemCount: Int) {
```

Comment:
Those assertions should be moved to another class named something like `BaristaListAssertions`. What do you think @Sloy ?